dev.lizardbyte.app.Sunshine.yml

```diff
@@ -0,0 +1,216 @@
+
+  - name: numactl
+    buildsystem: autotools
+    cleanup:
+      - "/bin"
+    sources:
+      - type: archive
+        url: https://github.com/numactl/numactl/archive/refs/tags/v2.0.16.tar.gz
+        sha256: a35c3bdb3efab5c65927e0de5703227760b1101f5e27ab741d8f32b3d5f0a44c
+        x-checker-data:
+          type: anitya
+          project-id: 2507
+          stable-only: true
+          url-template: https://github.com/numactl/numactl/archive/refs/tags/v$version.tar.gz
+
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://www.x.org/archive/individual/app/xrandr-1.5.2.tar.xz
+        sha256: c8bee4790d9058bacc4b6246456c58021db58a87ddda1a9d0139bf5f18f1f240
+        x-checker-data:
+          type: anitya
+          project-id: 14957
+          stable-only: true
+          url-template: https://www.x.org/archive/individual/app/xrandr-$version.tar.xz
+
+  - name: intel-mediasdk
+    only-arches:
+      - x86_64
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DBUILD_SAMPLES=OFF
+      - -DBUILD_TUTORIALS=OFF
+      - -DENABLE_X11=OFF
+    sources:
+      - type: archive
+        url: https://github.com/Intel-Media-SDK/MediaSDK/archive/refs/tags/intel-mediasdk-23.2.0.tar.gz
+        sha256: 6b33ec917ef0c6395fb66c7b93aae4ae31b3d1dedb5d7d39e0c69f45acb7c4fb
+        x-checker-data:
+          type: anitya
+          project-id: 21814
+          stable-only: true
+          url-template: https://github.com/Intel-Media-SDK/MediaSDK/archive/refs/tags/intel-mediasdk-$version.tar.gz
+
+  - name: cuda
+    buildsystem: simple
+    cleanup:
+      - "*"
+    build-commands:
+      - chmod u+x ./cuda.run
+      - ./cuda.run --silent --toolkit --toolkitpath=$FLATPAK_DEST/cuda --no-opengl-libs --no-man-page --no-drm
+        --tmpdir=$FLATPAK_BUILDER_BUILDDIR
+      - rm -r $FLATPAK_DEST/cuda/nsight-systems-*
+      - rm ./cuda.run
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://developer.download.nvidia.com/compute/cuda/12.1.1/local_installers/cuda_12.1.1_530.30.02_linux.run
+        sha256: d74022d41d80105319dfa21beea39b77a5b9919539c0487a05caaf2446d6a70e
+        dest-filename: cuda.run
+      - type: file
+        only-arches:
+          - aarch64
+        url: https://developer.download.nvidia.com/compute/cuda/12.1.1/local_installers/cuda_12.1.1_530.30.02_linux_sbsa.run
+        sha256: 45ea4cd860f0a26d3db8ce032530f2ee0b55abdd587545213d395a73623b4278
+        dest-filename: cuda.run
+
+  - name: sunshine
+    buildsystem: cmake
+    builddir: true
+    build-options:
+      append-path: /usr/lib/sdk/node18/bin
+      cxxflags: -I${C_INCLUDE_PATH}/libevdev-1.0
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_CUDA_COMPILER=/app/cuda/bin/nvcc
+      - -DSUNSHINE_ASSETS_DIR=share/sunshine
+      - -DSUNSHINE_EXECUTABLE_PATH="flatpak run --command=sunshine dev.lizardbyte.app.Sunshine"
```

Comment:
I will reiterate. flatpak do NO exist in the sandbox. That mean THIS IS WRONG. I still haven't explained what it is for (there is no systemd either).

Reply:
This is to give the option to configure as systemd user service . The user would copy it to ~/.config/systemd/user/ and enable it as a service.

Reply:
Result is
```
[Unit]
Description=@PROJECT_DESCRIPTION@
StartLimitIntervalSec=500
StartLimitBurst=5
PartOf=graphical-session.target
Wants=xdg-desktop-autostart.target
After=xdg-desktop-autostart.target

[Service]
ExecStart=flatpak run --command=sunshine dev.lizardbyte.app.Sunshine
Restart=on-failure
RestartSec=5s

[Install]
WantedBy=xdg-desktop-autostart.target
```

Reply:
not with the current sandbox permissions.

Reply:
also this is not the way auto starting in the user session is handled.